protokube/pkg/protokube/etcd_manifest.go

```diff
@@ -119,22 +101,108 @@ func BuildEtcdManifest(c *EtcdCluster) *v1.Pod {
 			},
 		})
 
-		container.VolumeMounts = append(container.VolumeMounts, v1.VolumeMount{
-			Name:      "varlogetcd",
-			MountPath: "/var/log/etcd.log",
-			ReadOnly:  false,
-		})
-		pod.Spec.Volumes = append(pod.Spec.Volumes, v1.Volume{
-			Name: "varlogetcd",
-			VolumeSource: v1.VolumeSource{
-				HostPath: &v1.HostPathVolumeSource{
-					Path: c.LogFile,
-				},
-			},
-		})
+		// @check if tls is enabled and mount the directory - it might be worth concidering
+		// if we you use our own directory in /srv i.e /srv/etcd
```

Comment:
That would make sense in my mind, in that we want to think about isolating all these keys from each other.  I think also using something under `/etc/kubernetes` would work well.

But I also believe not too hard to change later.